pkg/urbit/include/noun/options.h

```diff
@@ -9,6 +9,8 @@
         u3_noun who;                          //  single identity
         c3_c*   dir_c;                        //  execution directory (pier)
         c3_w    wag_w;                        //  flags (both ways)
+        size_t  byt_i;                        //  loom byte-length (<= u3a_bytes)
+        size_t  wor_i;                        //  loom word-length (<= u3a_words)
```

Comment:
Because these are redundant, I'd opt to remove `byt_i` and instead compute it using `wor_i` since `byt_i` only appears to be used in `u3e_yolo()`.

Reply:
Good point. I had included to avoid the question of overflow checks when shifting words to bytes, but it's always initialized from bytes so that's irrelevant.